boards/nullbitsco/nibble/main.py

```diff
@@ -0,0 +1,120 @@
+from kb import KMKKeyboard
+
+from kmk.extensions.media_keys import MediaKeys
+from kmk.extensions.rgb import RGB, AnimationModes
+from kmk.keys import KC
+
+keyboard = KMKKeyboard(encoder=True)  # assume encoder installed
+keyboard.extensions.append(MediaKeys())
+
+XXXXX = KC.NO
+
+keyboard.keymap = [
```

Comment:
Would you mind formatting this in the usual "looks like keymatrix" way? Prefix the keymap definition with `# fmt: off` and postfix with `# fmt:on` to selectively disable the linter.